p2p/host/relay/autorelay.go

```diff
@@ -48,12 +57,13 @@ type AutoRelay struct {
 	cachedAddrsExpiry time.Time
 }
 
-func NewAutoRelay(ctx context.Context, bhost *basic.BasicHost, discover discovery.Discoverer, router routing.PeerRouting) *AutoRelay {
+func NewAutoRelay(ctx context.Context, bhost *basic.BasicHost, discover discovery.Discoverer, router routing.PeerRouting, static []peer.AddrInfo) *AutoRelay {
 	ar := &AutoRelay{
 		host:       bhost,
 		discover:   discover,
 		router:     router,
 		addrsF:     bhost.AddrsFactory,
+		static:     static,
```

Comment:
@vyzo I know I'm late to the party here, but this felt a bit small for an issue.

Why did we bother changing the constructor and using a `[]Addr.Info` here instead of just creating a `Discoverer` implementation that returns a the static list?

Sure it can be more efficient to use the slice, but now we have this thing we're going to be stuck removing.